packages/client/hmi-client/src/components/workflow/scenario-templates/situational-awareness/tera-situational-awareness-template.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+	<div>
+		<h6>What's like to happen next?</h6>
```

Comment:
Is this an English idiome? I do not know this turn of phrase.

Reply:
Changed to "What's _likely_ to happen next?" 😄